desub-current/Cargo.toml

```diff
@@ -8,8 +8,8 @@ edition = "2018"
 [dependencies]
 log = "0.4"
 thiserror = "1.0.30"
-sp_core = { package = "sp-core",  git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12" }
-sp_runtime = { package = "sp-runtime", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12" }
+sp_core = { package = "sp-core",  git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
would be good pin to a certain commit or are we relying on `Cargo.lock` for that?